plugins/wme-sitebuilder/wme-sitebuilder/Wizards/FirstTimeConfiguration.php

```diff
@@ -169,6 +203,122 @@ public function isComplete() {
 		return (bool) $this->getData()->get( 'complete', false );
 	}
 
+	/**
+	 * Get the goal choices.
+	 *
+	 * @return array
+	 */
+	public function getGoalsChoices() {
+		$goals = [
+			[
+				'key'         => self::GOALS_BUSINESS,
+				'value'       => __( 'Create a home for myself, my business or organization online', 'wme-sitebuilder' ),
+				'description' => __( 'A web presence that shares information about my services, and expertise and helps customers find and get in contact with me.', 'wme-sitebuilder' ),
+				'icon'        => 'GraduationIcon', // @todo SVG String
```

Comment:
Need to match up each of the icons provided from the [Icons Directory](https://github.com/moderntribe/wme/blob/main/packages/sitebuilder/src/icons/index.ts) with the icon that matches it best.